modules/userId/index.js

```diff
@@ -124,6 +124,7 @@ import CONSTANTS from '../../src/constants.json';
 import { module, hook } from '../../src/hook.js';
 import { createEidsArray } from './eids.js';
 import { getCoreStorageManager } from '../../src/storageManager.js';
+import {getEidPermissions} from "./eids";
```

Comment:
Similar to the other comment, eids.js is defined above. Please combine as : 

`import  {createEidsArray, getEidPermissions } from './eids.js'`